docs/resize-single-disk-pool.md

```diff
@@ -0,0 +1,137 @@
+### 1. Do kubectl exec inside the `cstor-pool-mgmt` container using the following command and install `parted`
```

Comment:
Where will this file be included?

Reply:
Sorry. I am trying to understand the question. Do you mean the installed binary files? I didn't check it..

Reply:
got the answer. these are in a different branch. so am good.

Reply:
Correct..